lib/aruba/api.rb

```diff
@@ -206,6 +206,46 @@ def touch(*args)
       self
     end
 
+    # Copy a file and/or directory
+    #
+    # @param [String, Array] source
+    #   A single file or directory, multiple files or directories or multiple
+    #   files and directories. If multiple sources are given the destination
+    #   needs to be a directory
+    #
+    # @param [String] destination
+    #   A file or directory name. If multiple sources are given the destination
+    #   needs to be a directory
+    #
+    # rubocop:disable Metrics/CyclomaticComplexity
+    # rubocop:disable Metrics/PerceivedComplexity
+    def copy(*source, destination)
+      source = source.flatten
+
+      source.each do |s|
+        raise ArgumentError, %(The following source "#{s}" does not exist.) unless exist? s
+      end
+
+      raise ArgumentError, "Using a fixture as destination (#{destination}) is not supported" if destination.start_with? FIXTURES_PATH_PREFIX
+      raise ArgumentError, "Multiples sources can only be copied to a directory" if source.count > 1 && exist?(destination) && !directory?(destination)
```

Comment:
How does it behave if there's only one source but the destination is a file? Does it overwrite the file?